avaframe/out3Plot/outAIMEC.py

```diff
@@ -525,13 +535,14 @@ def resultVisu(cfgPath, cfgFlags, rasterTransfo, resAnalysis, plim):
             mk = mk+1
             if mk == len(markers):
                 mk = 0
-        ax1.legend(loc='lower left')
+        ax1.legend(loc=4)
 
     plt.xlim([-0.01, max(1, max(rFP))])
-    plt.ylim([0, 1.01])
+    plt.ylim([-0.01, 1.01])
```

Comment:
suggestion: still increase a tiny bit that the entire marker/symbol can be seen just for prettiness